tests/dragonfly/conftest.py

```diff
@@ -394,6 +394,8 @@ def pytest_runtest_makereport(item, call):
     if report.when == "teardown":
         log_dir = getattr(item, "log_dir", None)
         call_outcome = getattr(item, "call_outcome", None)
+        if report.failed:
```

Comment:
Can you plz add a commit that verifies this ? Basically I would like you to test that this work. You can revert this afterwards. 

Also if you could add a small description on the PR it would be perfect

Reply:
I test it on my PC, but if you want I can do it here too

Reply:
Just in case, you never know with workflows